src/node_crypto.h

```diff
@@ -488,22 +488,21 @@ class Hmac : public BaseObject {
 
   static void Initialize(Environment* env, v8::Local<v8::Object> target);
 
- protected:
   void HmacInit(const char* hash_type, const char* key, int key_len);
   bool HmacUpdate(const char* data, int len);
   bool HmacDigest(unsigned char** md_value, unsigned int* md_len);
-
-  static void New(const v8::FunctionCallbackInfo<v8::Value>& args);
-  static void HmacInit(const v8::FunctionCallbackInfo<v8::Value>& args);
-  static void HmacUpdate(const v8::FunctionCallbackInfo<v8::Value>& args);
-  static void HmacDigest(const v8::FunctionCallbackInfo<v8::Value>& args);
-
```

Comment:
I'm not sure about these changes. Not only would it create an inconsistency with the other crypto C++ APIs, but I *think* doing so could even constitute a `semver-major` change?